ci/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM debian:buster-slim
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+ENV PATH /opt/conda/bin:$PATH
+
+# hadolint ignore=DL3008
```

Comment:
What's that?

Reply:
A Dockerfile linker https://github.com/hadolint/hadolint

Reply:
Ah, neat, we should add that.

Reply:
added :)